framework/meta/src/cmd/contract/sc_config/wasm_build.rs

```diff
@@ -37,6 +38,9 @@ impl ContractVariant {
         }
         if let Some(target_dir_wasm) = &build_args.target_dir_wasm {
             command.args(["--target-dir", target_dir_wasm]);
+        } else {
```

Comment:
else if let Some(workspace) = find_current_workspace() {...

Reply:
This is where you can print something
Like ...
"Using workspace target directory: {}"